ui-react/packages/atlasmap-ui/src/models.ts

```diff
@@ -6,7 +6,7 @@ export type Rect = { id: string, x: number, y: number, width: number, height: nu
 export type Rects = Array<Rect>;
 export type BrowserRect = ClientRect | DOMRect;
 export type ElementId = string;
-export type ElementType = 'source' | 'target';
+export type DocumentType = 'source' | 'target';
```

Comment:
DocumentType should probably be reserved for JSON, XML XSD, JAVA, etc.  This is really the 'DocumentPanel'

Reply:
Agree, I'll clean this up in the next PR where I'm moving the rendering of the fields text and related events up to the `Atlasmap` component.